Src/Support/Google.Apis.Auth/OAuth2/DefaultCredentialProvider.cs

```diff
@@ -147,15 +147,12 @@ private async Task<GoogleCredential> CreateDefaultCredentialAsync()
                     HelpPermalink));
         }
 
-        /// <summary>Creates a default credential from a JSON file.</summary>
-        private GoogleCredential CreateDefaultCredentialFromFile(string credentialPath)
+        private async Task<GoogleCredential> CreateDefaultCredentialFromFileAsync(string credentialPath, CancellationToken cancellationToken)
         {
-            Logger.Debug("Loading Credential from file {0}", credentialPath);
+            Logger.Debug("Aync loading Credential from file {0}", credentialPath);
```

Comment:
Revert this change, after you've tested that it's working? (It would at least need fixing to Async, but I don't think that's really necessary.)

Reply:
Ah yes, will do.

Also though about adressing the TODO, we have a bit of duplicate code because of it. But I'll defer that for the near future to do carefully.